library/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellStepsDefinitionProvider.java

```diff
@@ -61,82 +63,63 @@ public List<DefinitionVersion> availableVersions() {
   @Override
   public List<Pair<Step, RetryRule>> get(
```

Comment:
The changes here aren't _as_extensive as they appear. I moved the `GetManagedResourceGroupInfo` step to the top since it's needed to generate the region-specific `ParametersResolver`. Then I added the `GetParametersResolver` step which creates the region-specific `ParametersResolver` and adds it to the flight's working map. The `ParametersResolver` is no longer passed to each step as a parameter, the steps get it out of the working map